src/lightning/fabric/accelerators/cuda.py

```diff
@@ -181,62 +181,168 @@ def is_cuda_available() -> bool:
 
 
 # TODO: Remove once minimum supported PyTorch version is 1.13
```

Comment:
```suggestion
# TODO: Remove once minimum supported PyTorch version is 2.0
```

Reply:
L164 now needs to check for 2_0

Reply:
No, why? If every time there is a fix we need to bump the version, we will never be able to drop the code and rely on pytorch.

Reply:
These fixes that you are incorporating are only part of PyTorch 2.0

So we can only rely on pytorch after that. This is when our minimum supported PyTorch version is 2.0

Or am I missing something?